frontend/src/components/InlineNotification/index.js

```diff
@@ -10,6 +10,7 @@ const CustomInlineNotification = styled(Inline)`
   bottom: 20px;
   right: 20px;
   position: fixed;
+  z-index: 2000;
```

Comment:
I think this is too much. It looks better when the notification doesn't appear on top of the webpage